### PR TITLE
Added row of control buttons to navigate frames, play, and clear

### PR DESCRIPTION
**Pull request by drinkthestars**
Left a `TODO` to implement the "Play" button

### CODE REVIEW

File: draw_flip_book/lib/main.dart [hunk 1]
```diff
@@ -79,31 +79,83 @@ class _FlipBookPageState extends State<FlipBookPage> {
     // fast, so that you can just rebuild anything that needs updating rather
     // than having to individually change instances of widgets.
     return new Scaffold(
```
Comment (maryx) [inline draw_flip_book/lib/main.dart]: Can remove `new`

File: draw_flip_book/lib/main.dart [hunk 2]
```diff
@@ -79,31 +79,83 @@ class _FlipBookPageState extends State<FlipBookPage> {
...
-            setState(() {
-              _toggleFramesVisibility();
-            });
-          },
-          child: Icon(Icons.skip_next)),
     );
   }
 
-  GestureDetector buildGestureDetector(BuildContext context, Widget child) {
+  Row _buttonRow() {
```
Comment (maryx) [inline draw_flip_book/lib/main.dart]: In general, prefer to just return Widget type.

File: draw_flip_book/lib/main.dart [hunk 3]
```diff
@@ -79,31 +79,83 @@ class _FlipBookPageState extends State<FlipBookPage> {
...
+    return Row(
```
Comment (maryx) [inline draw_flip_book/lib/main.dart:137]: Maybe break the containers in the children down to vars. So we know what each button does too.

File: draw_flip_book/lib/main.dart [hunk 4]
```diff
@@ -79,31 +79,83 @@ class _FlipBookPageState extends State<FlipBookPage> {
...
+            },
+            child: Icon(Icons.clear),
+          ),
+        ),
+      ],
+    );
+  }
+
+  Stack _framesStack(BuildContext context) {
+    return Stack(
```
Comment (maryx) [inline draw_flip_book/lib/main.dart]: For simple members, consider using => (not picky though).
https://dart.dev/guides/language/effective-dart/usage#consider-using--for-simple-members

Comment (drinkthestars) [reply draw_flip_book/lib/main.dart]: nice, 8db9d38

File: draw_flip_book/lib/main.dart (hunk 3 above)
Comment (drinkthestars) [reply draw_flip_book/lib/main.dart:137]: :+1: 6c0306d

File: draw_flip_book/lib/main.dart (hunk 2 above)
Comment (drinkthestars) [reply draw_flip_book/lib/main.dart]: ahh yes b1cc124

File: draw_flip_book/lib/main.dart (hunk 1 above)
Comment (drinkthestars) [reply draw_flip_book/lib/main.dart]: 👍 b1cc124

**Comment by maryx** [comment]
LGTM

**Pull request merged**